sr_moveit_hand_config/scripts/sr_moveit_hand_config/generate_moveit_config.py

```diff
@@ -32,8 +32,8 @@
 
 """
 generate_moveit_config provides:
-    generate_fake_controllers : generage a fake controllers config file
-    generate_real_controllers : generage a controllers config file
+    generate_fake_controllers : generate a fake controllers config file
+    generate_real_controllers : generate a controllers config file
     generate_ompl_planning : generate ompl_planning config file
```

Comment:
all the others don't have underscores between the words, for consistency, should be 'ompl planning'?